dev/create-release/spark-rm/Dockerfile

```diff
@@ -62,14 +62,18 @@ RUN apt-get clean && apt-get update && $APT_INSTALL gnupg ca-certificates && \
   curl -sL https://deb.nodesource.com/setup_11.x | bash && \
   $APT_INSTALL nodejs && \
   # Install needed python packages. Use pip for installing packages (for consistency).
-  $APT_INSTALL libpython2.7-dev libpython3-dev python-pip python3-pip && \
-  pip install $BASE_PIP_PKGS && \
-  pip install $PIP_PKGS && \
+  $APT_INSTALL libpython3-dev python3-pip && \
+  # Change default python version to python3.
+  update-alternatives --install /usr/bin/python python /usr/bin/python2.7 1 && \
+  update-alternatives --install /usr/bin/python python /usr/bin/python3.6 2 && \
+  update-alternatives --set python /usr/bin/python3.6 && \
+  pip3 install $BASE_PIP_PKGS && \
+  pip3 install $PIP_PKGS && \
   cd && \
-  virtualenv -p python3 /opt/p35 && \
+  python /opt/p35 && \
```

Comment:
`No such file or directory` when building docker image:
```
Successfully built pypandoc pycparser
Installing collected packages: six, pycparser, cffi, cryptography, pyopenssl, pypandoc, numpy, Pygments, sphinxcontrib-applehelp, pytz, babel, snowballstemmer, imagesize, urllib3, certifi, chardet, idna, requests, sphinxcontrib-serializinghtml, sphinxcontrib-devhelp, sphinxcontrib-htmlhelp, alabaster, sphinxcontrib-jsmath, sphinxcontrib-qthelp, pyparsing, packaging, MarkupSafe, Jinja2, docutils, sphinx
Successfully installed Jinja2-2.10.3 MarkupSafe-1.1.1 Pygments-2.5.2 alabaster-0.7.12 babel-2.7.0 certifi-2019.11.28 cffi-1.13.2 chardet-3.0.4 cryptography-2.8 docutils-0.15.2 idna-2.8 imagesize-1.1.0 numpy-1.17.4 packaging-19.2 pycparser-2.19 pyopenssl-19.1.0 pypandoc-1.4 pyparsing-2.4.5 pytz-2019.3 requests-2.22.0 six-1.13.0 snowballstemmer-2.0.0 sphinx-2.2.2 sphinxcontrib-applehelp-1.0.1 sphinxcontrib-devhelp-1.0.1 sphinxcontrib-htmlhelp-1.0.2 sphinxcontrib-jsmath-1.0.1 sphinxcontrib-qthelp-1.0.2 sphinxcontrib-serializinghtml-1.1.3 urllib3-1.25.7
python: can't open file '/opt/p35': [Errno 2] No such file or directory
```

Reply:
@cloud-fan Can we remove `virtualenv`?

Reply:
I think you can remove ` python /opt/p35 && /opt/p35/bin/activate`.

Reply:
That seems in order to activate virtualenv.

Reply:
Can we remove this?
https://github.com/apache/spark/blob/8d1b5ba76651c049c957ad48ae60f2c8656dff49/dev/create-release/do-release-docker.sh#L139-L140

Reply:
To remove that out, we should revert https://github.com/apache/spark/commit/1f94bf492c3bce3b61f7fec6132b50e06dea94a8

I think you can set that as `python`, `python3` or `python3.6` explicitly for now.

Reply:
You can revert https://github.com/apache/spark/commit/1f94bf492c3bce3b61f7fec6132b50e06dea94a8 here and remove that too because we don't seem to use Python 2 anymore in dev. cc @shaneknapp

If you want, you can do it separately too. We should also update release guide if we will remove that one. See https://github.com/apache/spark-website/pull/122

Reply:
Thank you @HyukjinKwon . We need to do it:
```
= Building documentation...
Command: /opt/spark-rm/release-build.sh docs
Log file: docs.log
Command FAILED. Check full logs for details.
Copying jquery.min.js from Scala API to Java API for page post-processing of badges
Copying api_javadocs.js to Java API for page post-processing of badges
Appending content of api-javadocs.css to JavaDoc stylesheet.css for badge styles
Moving to python/docs directory and building sphinx.
make: /opt/p35/bin/python: Command not found
/opt/p35/bin/python -msphinx -b html -d _build/doctrees   . _build/html
make: /opt/p35/bin/python: Command not found
Makefile:80: recipe for target 'html' failed
make: *** [html] Error 127
jekyll 3.8.6 | Error:  Python doc generation failed
```

Reply:
> `make: /opt/p35/bin/python: Command not found`
Can we try to set `SPHINXPYTHON=python` and see if it works? Seems it's just because `python` was not found.

Reply:
I reverted commit 1f94bf4 and d6be46e, I'd like to to confirm is it correct??
![image](https://user-images.githubusercontent.com/5399861/70693075-a7723100-1cf7-11ea-9f88-9356a02349a1.png)

Reply:
Yeah. Seems right to revert two commits (and push both reverts into this PR).

Reply:
cc @cloud-fan.